BlueprintUI/Sources/Blueprint View/BlueprintView.swift

```diff
@@ -141,11 +141,14 @@ public final class BlueprintView: UIView {
         
         rootController.view.frame = bounds
         
-        let rootNode = NativeViewNode(
+        var rootNode = NativeViewNode(
             content: UIView.describe() { _ in },
             layoutAttributes: LayoutAttributes(frame: bounds),
             children: viewNodes
         )
+
+        let scale = window?.screen.scale ?? UIScreen.main.scale
```

Comment:
Should we do `setNeedsViewHierarchyUpdate` in `didMoveToWindow`? (a view moving from a window at one scale to a window at another is _entirely_ an edge case, but…)

Reply:
Yeah, I suppose. (If that did happen we would also have problems with elements that do measuring with prototype view instances.)